test/unit/jasmine-boot.js

```diff
@@ -43,58 +43,25 @@
-    function (sharedUtil, displayGlobal, corePrimitives, coreAnnotation,
-              coreCrypto, coreStream, coreFonts, corePsParser, coreFunction,
-              coreParser, coreEvaluator, coreCMap, coreWorker, coreNetwork,
-              coreType1Parser, coreCFFParser, coreMurmurHash3, displayAPI,
-              displayMetadata, displayDOMUtils, webUIUtils, coreUnicode,
-              coreGlyphList) {
-
-      pdfjsLibs = {
-        sharedUtil: sharedUtil,
-        displayGlobal: displayGlobal,
-        corePrimitives: corePrimitives,
-        coreAnnotation: coreAnnotation,
-        coreCrypto: coreCrypto,
-        coreStream: coreStream,
-        coreFonts: coreFonts,
-        corePsParser: corePsParser,
-        coreFunction: coreFunction,
-        coreParser: coreParser,
-        coreEvaluator: coreEvaluator,
-        coreCMap: coreCMap,
-        coreWorker: coreWorker,
-        coreNetwork: coreNetwork,
-        coreType1Parser: coreType1Parser,
-        coreCFFParser: coreCFFParser,
-        coreMurmurHash3: coreMurmurHash3,
-        displayAPI: displayAPI,
-        displayMetadata: displayMetadata,
-        displayDOMUtils: displayDOMUtils,
-        webUIUtils: webUIUtils,
-        coreUnicode: coreUnicode,
-        coreGlyphList: coreGlyphList,
-      };
-
-      // Expose all loaded internal exported members to global scope.
-      Object.keys(pdfjsLibs).forEach(function (libName) {
-        var lib = pdfjsLibs[libName];
-        Object.keys(lib).forEach(function (name) {
-          if (Object.getOwnPropertyDescriptor(window, name)) {
-            return; // ignoring if already set
-          }
-          window[name] = lib[name];
-        });
-      });
+  require.config({paths: {'pdfjs': '../../src', 'pdfjs-web': '../../web',
+    'pdfjs-test': '..'}});
+  require(['pdfjs/display/global', 'pdfjs-test/unit/annotation_layer_spec',
+    'pdfjs-test/unit/api_spec', 'pdfjs-test/unit/cff_parser_spec',
+    'pdfjs-test/unit/cmap_spec', 'pdfjs-test/unit/crypto_spec',
+    'pdfjs-test/unit/dom_utils_spec', 'pdfjs-test/unit/evaluator_spec',
+    'pdfjs-test/unit/fonts_spec', 'pdfjs-test/unit/function_spec',
+    'pdfjs-test/unit/metadata_spec', 'pdfjs-test/unit/murmurhash3_spec',
+    'pdfjs-test/unit/network_spec', 'pdfjs-test/unit/parser_spec',
+    'pdfjs-test/unit/primitives_spec', 'pdfjs-test/unit/stream_spec',
+    'pdfjs-test/unit/type1_parser_spec', 'pdfjs-test/unit/ui_utils_spec',
+    'pdfjs-test/unit/unicode_spec', 'pdfjs-test/unit/util_spec'],
+    function (displayGlobal, testUnitAnnotationLayerSpec, testUnitApiSpec,
+      testUnitCFFParserSpec, testUnitCMapSpec, testUnitCryptoSpec,
+      testUnitDOMUtilsSpec, testUnitEvaluatorSpec, testUnitFontsSpec,
+      testUnitFunctionSpec, testUnitMetadataSpec, testUnitMurmurHash3Spec,
+      testUnitNetworkSpec, testUnitParserSpec, testUnitPrimitivesSpec,
+      testUnitStreamSpec, testUnitType1ParserSpec, testUnitUiUtilsSpec,
+      testUnitUnicodeSpec, testUnitUtilSpec) {
```

Comment:
Nit: Please change the indentation of this line so that the parameter names line up, compare with e.g. [this file](https://github.com/mozilla/pdf.js/blob/aabfb7788ad140c424dd0b7823f35255d13f5225/src/display/api.js#L35-L36); it should look like this:
```javascript
function (displayGlobal, testUnitAnnotationLayerSpec, testUnitApiSpec,
          testUnitCFFParserSpec, testUnitCMapSpec, testUnitCryptoSpec,
          testUnitDOMUtilsSpec, testUnitEvaluatorSpec, testUnitFontsSpec,
          testUnitFunctionSpec, testUnitMetadataSpec, testUnitMurmurHash3Spec,
          testUnitNetworkSpec, testUnitParserSpec, testUnitPrimitivesSpec,
          testUnitStreamSpec, testUnitType1ParserSpec, testUnitUiUtilsSpec,
          testUnitUnicodeSpec, testUnitUtilSpec) {
```